convert2rhel/toolopts.py

```diff
@@ -79,7 +79,7 @@ def __init__(self):
     def _get_argparser():
         usage = (
             "\n"
-            "  convert2rhel [-h]\n"
+            "  convert2rhel [-h]<n>"
```

Comment:
Is this correct?

Reply:
Oh, I must have changed it by mistake while tyring out a few things. Thanks for spotting that out! 

```suggestion
            "  convert2rhel [-h]\n"
```